engine/runtime/src/main/java/org/enso/interpreter/runtime/data/vector/ArraySlice.java

```diff
@@ -176,12 +176,12 @@ boolean isLimitReached(@Shared("warnsLib") @CachedLibrary(limit = "3") WarningsL
   }
 
   @ExportMessage
-  boolean hasType(@Shared("ignore") @Cached("1") int ignore) {
```

Comment:
Great. The upgrade pays off just by removing these `@Cached ignore` workarounds.

Reply:
Please remove also those 25 other usages of this workaround.

Reply:
Removed the rest of the workarounds in 985a9cb6d0c860bca7e49ca058ef886a1b029c1e